### ISSUE TITLE
Windows service doesn't work using ProgramData config directory

### ISSUE THREAD

**Issue by lucknaumann** [morpheus65535/bazarr#1682]
**Describe the bug**
I updated manually a couple of weeks ago with the installer since my Bazarr install wasn't working. I had to run the bazarr.py program manually through cmd.exe because it still wasn't working. After a reboot (figuring that might be the issue), it still wasn't working automatically, even though the Windows service was running. When attempting to run bazarr through cmd.exe with the flags that nssm.exe had set through the installer, I noticed that the ProgramData config file had an issue with permissions (it needs to run as admin, even though I was using an admin console). I used nssm.exe to remove the `--config` flag from the Windows service, and it now works (using the `C:\Bazarr\data` directory). 

**To Reproduce**
1. Install Bazarr using the Windows installer.
2. Go to http://localhost:6767/
3. See the error page that it can't load.
4. 

**Expected behavior**
For the bazarr Windows service to work, allowing the page to load at http://localhost:6767/

**Screenshots**

**Software (please complete the following information):**
 - Bazarr: 1.0.2
 - Radarr version 4.0.1.5813
 - Sonarr version 3.0.6.1342
 - OS: Windows 10 21H2

**Additional context**

**Comment by WeetbixNMilk** [comment]
+1 I am also having this issue when installing to C:/User/ProgramData so it's alongside Sonarr, Radarr, etc.

Upon a restart of my PC, my database, web interface and service all have been affected.

@lucknaumann Was your only fix to change directory and config?

**Comment by morpheus65535** [comment]
> +1 I am also having this issue when installing to C:/User/ProgramData so it's alongside Sonarr, Radarr, etc.
> 
> Upon a restart of my PC, my database, web interface and service all have been affected.

Although undocumented, installing to %programdata%/Bazarr isn't a good idea because everytime Bazarr will update, the leftover cleanup routine (responsible to remove from your installation directory the files that are no longer needed) will also mark for deletion the config files that are in the exact same directory.

It's a known issue that we'll try to fix in the upcoming release but for now the best solution is to install somewhere else to let %programdata%/Bazarr directory only for config files.

**Comment by morpheus65535** [comment]
@lucknaumann your issue is simply that your new Bazarr service is running with an account that doesn't have access to the previously created config directory. You can either:

1. change the user running Bazarr to match your previous installation
2. change permission recursively on the %programdata%/Bazarr directory to give access to your actual user
3. simply delete the %programdata%/Bazarr directory and let Bazarr recreate a new one on which it's gonna have full access.

The nssm modification you did won't hurt, it was just not necessary.

**Comment by lucknaumann** [comment]
[**@morpheus65535**](https://github.com/morpheus65535) commented on [Jan 14, 2022, 5:12 AM MST](https://github.com/morpheus65535/bazarr/issues/1682#issuecomment-1013066783 "2022-01-14T12:12:11Z - Replied by Github Reply Comments"):
> [@lucknaumann](https://github.com/lucknaumann) your issue is simply that your new Bazarr service is running with an account that doesn't have access to the previously created config directory. You can either:
> 
> 1.  change the user running Bazarr to match your previous installation
> 2.  change permission recursively on the %programdata%/Bazarr directory to give access to your actual user
> 3.  simply delete the %programdata%/Bazarr directory and let Bazarr recreate a new one on which it's gonna have full access.
> 
> The nssm modification you did won't hurt, it was just not necessary.

Thanks for your speedy response.

Yes, my account does have access to everything and everywhere as the Windows default user accounts are Admin. I'm the only one using this machine, and my account is, therefore, an Admin account. 

I did delete the `%ProgramData%\Bazarr` directory, and Bazarr decided to use the `C:\Bazarr\data` directory. Prior to doing that, I made sure the Bazarr folder and the files within had the necessary permissions for my account to access the files and opened the config file in there without issue in notepad.exe. Since it still wouldn't work, I copied the corresponding files in the data folder to `%ProgramData%\Bazarr`, replacing the files in there. It still wouldn't work. Only after removing the config flag from the Windows service did it pick up the  `C:\Bazarr\data` directory and worked.

**Comment by morpheus65535** [comment]
@lucknaumann actually you may have some additional enforcement or limitation due to admin account usage. The actual Windows installer is working for 1800 users during the last week alone. I would be surprise if they all run into the same issue and not open an issue...
